ui/templates/base_error.html

```diff
@@ -22,11 +22,12 @@
 	  		{% if authenticated %}
 	  		<div class="dropdown btn-group">
               <a class="btn btn-danger" href="/dashboard"><span>{{ name }}</span></a>
```

Comment:
the errors are served without using REACT because they are common to the CMS and made with templates only.
The `base_error.html` is the common base template that extends `base.html` and it is used by `404.html` and `500.html`